lessons/09-clientside-frameworks/INCLASS.md

```diff
@@ -1,27 +1,37 @@
 
-##Virtual DOM
-React uses VDOM, which results in much better performance than Angular rendering views. As thoroughly discussed [here](https://facebook.github.io/react/docs/glossary.html), ReactElements and their properties allow the user to alter elements using the virtual DOM. As noted, *a ReactElement is a light, stateless, immutable, virtual representation of a DOM Element*. 
+##Research
+- Find Angular and React documentation online. Pay particular attention to the following keywords:
 
-##Binding	
-Let's start with data binding. In Angular, one can only bind to scope. This essentially means that in complex scenarios, such as binding to a server, one will require an intermediate model in the way, as well as, one will have to take into account the handling of digest cycles in the process. On the other hand, React's beauty provides us with a *platform* for binding. A single attribute that takes into account both the *value* and the *onChange* properties of an object; namely, valueLink. For more information on data binding checkout the links for the respective framework:
-- [React Data Binding](https://facebook.github.io/react/docs/two-way-binding-helpers.html)
-- [Angular Data Binding](https://docs.angularjs.org/guide/databinding)
-
-##JSX
-In essence, JSX is a preprocessor step that adds XML syntax to JavaScript. One can definitely use React without JSX but JSX makes React a lot more elegant. JSX tags have a tag name, attributes and nested children. In addition to the above, XML has the benefit of balanced opening and closing tags. This helps make large trees easier to read than function calls or object literals. For more information on JSX, visit [here](https://facebook.github.io/react/docs/jsx-in-depth.html).
+####Angular
+- Two-way data binding
+- Controllers
+- Services
+- Directives
+- Scopes
 
-##Which one to use?
-It is important to note that React can't stand as a framework on its own and *can* be wrapped in Angular. As afore-mentioned, there are both pros and cons in the use of either framework. An interesting read comparing Angular and React can be further found [here](https://medium.freecodecamp.com/angular-2-versus-react-there-will-be-blood-66595faafd51).
+####React
+- Virtual DOM
+- ReactComponents
+- JSX
+- State vs Props
+- Uni-directional Data flow
+- Flux
 
+Here are some links to start you out:
+- [Angular Conceptual Overview](https://docs.angularjs.org/guide/concepts)
+- [Angular Data Binding](https://docs.angularjs.org/guide/databinding)
+- [Why Build React?](http://facebook.github.io/react/blog/2013/06/05/why-react.html)
+- [A Closer Look into React by Pete Hunt](https://www.quora.com/profile/Pete-Hunt/Posts/Facebooks-React-vs-AngularJS-A-Closer-Look)
+- [JSX](https://facebook.github.io/react/docs/jsx-in-depth.html)
 
-##Hello Angular 1.0
+##Hello Angular
 In it's simplest form, one can use the code provided below to asynchronously update and place the content from the textbox to complete the period next to "Hello".
```

Comment:
Could this be: 
Here's just about the simplest Angular app you can write. The words you type in the textbox will show up after "Hello" in the `<h1>` tag.